<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="syntax">
    <attribute name="title">
     <string>Syntax</string>
    </attribute>
    <widget class="QCheckBox" name="syntax_bioperator">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>341</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Add blanks beside bioperators</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="syntax_leftBraceNewLine">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Keep left brace in the front of a new line</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="syntax_spaceAfterComma">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Add a space after each comma</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="syntax_alwaysQuoteBlocks">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Always quote blocks with braces</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="syntax_splitFunctions">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Split functions with blank lines</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="syntax_noComments">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Do not erase the comments</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QRadioButton" name="nomenclature_camelCase">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Camel Case</string>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string>syntax_nomenclature</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="nomenclature_underline">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Underline-splitted</string>
     </property>
     <attribute name="buttonGroup">
      <string>syntax_nomenclature</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="nomenclature_pascal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pascal</string>
     </property>
     <attribute name="buttonGroup">
      <string>syntax_nomenclature</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="nomenclature_hungary">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>190</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Hungary</string>
     </property>
     <attribute name="buttonGroup">
      <string>syntax_nomenclature</string>
     </attribute>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nomenclature</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="nomenclature_skipUpperCase">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Skip Upper-case identifiers</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="global">
    <attribute name="title">
     <string>Global</string>
    </attribute>
    <widget class="QCheckBox" name="displayToolBarOnStart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Display Popup Tool Bar on start</string>
     </property>
    </widget>
    <widget class="QComboBox" name="displayLanguage">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string notr="true" comment="en_US">English</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true" comment="zh_CN">中文(中国)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true" comment="ja_JP">日本語(日本)</string>
      </property>
     </item>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Display Language</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="syntax_nomenclature"/>
 </buttongroups>
</ui>
